frontend/src/Interfaces/DialogsProps.tsx

```diff
@@ -0,0 +1,17 @@
+import { MouseEventHandler } from "react";
+
+export interface AlertDialogProps {
+  title: string | undefined;
```

Comment:
shouldn't this be always a `string`?

Reply:
compulsory fields shouldn't be nullable

plus, nullable field should be indicated with:
```
export interface SomeProps {
  nullableField?: string
}
```